misc/estimate_cost.test.js

```diff
@@ -16,7 +16,7 @@ async function originateWarehouse() {
     const originationOp = await Tezos.contract.originate({
```

Comment:
the files in the `misc` folder are just a bunch of scripts I use for testing various aspects of the contracts. This is not production code, just test snippets